### ISSUE TITLE
question on -level behaviour

### ISSUE THREAD

**Issue by pgajdos** [ImageMagick/ImageMagick#1132]
-level does not have the effect described in
http://www.imagemagick.org/Usage/color_mods/
e. g.:
```
convert  rose:  -level 100%,0  rose_level_neg.gif
convert  rose:  -level 50%,50%  rose_level_thres.gif
```
for me with 6.9.9-43 and 7.0.7-29. I probably missed something from times of ImageMagick-6.8.8-1, where it seems to work as expected. Could you please point to the right direction?

**Comment by fmw42** [comment]
Using both ImageMagick 6.9.9.44 Q16 and 7.0.7.32 Q16 HDRI, I find that -level 100%,0 works fine for me. But -level 50%,50% does not. I compared the former with -negate and get the same results.

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in GIT master branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://www.imagemagick.org/download/beta/ by sometime tomorrow.

**Comment by pgajdos** [comment]
Thank you.

As far as I tested, the issue is fixed in 7.0.7-34 and 6.9.9-46.
